cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/Checker.java

```diff
@@ -50,15 +49,15 @@ public void accept(CommandInvocationNode node) {
 
                     ArgumentNode argument = (ArgumentNode) setter.apply(node);
 
-                    if (argument == null || argument.getArgument().startsWith("CMAKE_")) {
+                    if (argument == null || SimulationState.getArgument(argument).startsWith("CMAKE_")) {
                         return;
                     }
 
                     suspiciousPoints.add(node);
                 }
             });
 
-            AbstractSimulator unusedSimulator = new AbstractSimulator(suspiciousPoints) {
+            AbstractSimulator unusedSimulator = new AbstractSimulator() {
                 @Override
                 protected void init(Map<String, CommandSimulator> simulators) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) The Cyclomatic Complexity of this method "init" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)